wordpress-mobile/WordPress-iOS
# [Gutenberg] React-navigation in bottom-sheet

In this PR I added a `react-navigation` to use in the bottom-sheet. The react-navigation requires a few native modules to be linked so i added them as well.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/23782
Gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2549

To test:

- Run editor
- Check settings of each block
- Open Button Block settings
- Click on Text Color / Backgrounds color
- The navigation inside bottom sheet should be possible
- Play a bit with colors and check if the button reflecting these changes
- Try the hardware back button on Android - the back button should back to the previous screen or hide the bottom-sheet if it's the main settings screen

PR submission checklist:
